test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -330,6 +330,10 @@ Describe "New-Item: symlink with absolute/relative path test" -Tags @('CI', 'Req
         $link = New-Item -Type SymbolicLink globbedAbsoluteLink1 -Target $absolutePath
         (Get-Item $link).Target | Should -BeExactly $absolutePath
 
+        if (-not $IsWindows) {
+            Set-ItResilt -Pending -Because "On Unix the Target property is always resolved to an absolute path."
+        }
```

Comment:
If we need to mark this `It` as pending, we may want to separate the assertions into two separate tests.

Otherwise, the result will be confusing from this test (it might fail sometimes, but at most it will only be Pending).

Also, the cmdlet name is misspelled here. 🙂

Reply:
@vexx32 Thanks! I split tests.